src/app.html

```diff
@@ -4,8 +4,13 @@
 		<meta charset="utf-8" />
 		<link rel="icon" href="%sveltekit.assets%/favicon.png" />
 		<link rel="manifest" href="%sveltekit.assets%/manifest.json" crossorigin="use-credentials" />
-		<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2" />
```

Comment:
Shouldn't be changed.

Reply:
What's the reasoning behind this?

https://dequeuniversity.com/rules/axe/4.9/meta-viewport